components/aws/sagemaker/batch_transform/src/sagemaker_transform_component.py

```diff
@@ -0,0 +1,168 @@
+"""SageMaker component for transform."""
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
check if this is the only header, I think we are missing a amazon copyright header as well

Reply:
True. I will go through and add copyright to each file. I'll save it for now because it will touch every file so it will be harder to review.